webpack/scenes/ActivationKeys/Details/components/DeleteModal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {
+  useDispatch,
+} from 'react-redux';
+import PropTypes from 'prop-types';
+import { noop } from 'foremanReact/common/helpers';
+import { Modal, ModalVariant, Button, Icon, Title, Flex } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { deleteActivationKey } from '../ActivationKeyActions';
+
+const DeleteModal = ({ isModalOpen, handleModalToggle, akId }) => {
+  const dispatch = useDispatch();
+
+  const handleDelete = () => {
+    dispatch(deleteActivationKey(akId));
+    handleModalToggle();
+    window.location.replace('/activation_keys');
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
<> = <React.Fragment>
For this component, you could directly return <Modal> since there aren't any other child components in the <>.